packages/lib/src/components/Dropin/types.ts

```diff
@@ -101,14 +101,25 @@ export interface DropinConfiguration extends UIElementProps {
 }
 
 export interface onOrderCancelData {
-    order: Order;
+    order: {
+        orderData: string;
+        pspReference: string;
+    };
 }
 
+export type onOrderCancelType = (
+    data: onOrderCancelData,
+    actions: {
+        resolve: (data: { amount: PaymentAmount }) => void;
+        reject: (error: string) => void;
```

Comment:
Not sure if we discussed this, but does it make sense to pass an error string here? what will happen in this case? is it just forwarded back as `AdyenCheckoutError` ?